src/core/configuration/index.ts

```diff
@@ -33,5 +32,5 @@ export const getDeviceConf = (
 };
 
 export default {
-  search
+  dplConfig
```

Comment:
The idea was to put the different configuration scopes into this: search, colors, modal-ids, status-threshold. And have one way of fetching the configuration. We have now two ways of doing it. And calling the scope `dplConfig` is, in my eyes, the same as calling the scope "stuff" or "whatever".
So in my view the right solution here is getting all the configuration here and use the same way of retrieving the configuration.
Regarding the comment about that cover tints is also defined along the pager settings could be solved with separating the two types of configuration into the scopes: "pager" and "covers"